pyuvdata/tests/test_miriad.py

```diff
@@ -67,6 +67,8 @@ def test_ReadMiriadWriteUVFits():
     miriad_uv = UVData()
     uvfits_uv = UVData()
     miriad_file = os.path.join(DATA_PATH, 'zen.2456865.60537.xy.uvcRREAA')
+    if not os.path.exists(os.path.join(DATA_PATH, 'test')):
+        os.mkdir(os.path.join(DATA_PATH, 'test'))
```

Comment:
This is done in the `setup_package` function in `tests/__init__.py` so this shouldn't be necessary here. Was this fixing a particular problem?

Reply:
Yeah, this was failing for me locally. I guess for some reason my setup wasn't doing this for me? Anyways, doesn't hurt I guess to have this..

Reply:
hmm.. I want to understand what's going on. I'll check out the branch and see whether I have the same problem. Were you testing on python 2 or 3 when you had the problem?

Reply:
When I comment out those lines and run `nosetests pyuvdata/tests/test_miriad.py` I don't get an error in either python 2 or 3, but I do get a warning that's not being caught...

Reply:
This was testing for python2. So  I don't get the error when I also run `nosetest test_miriad.py`, but if I enter an interactive session and try to run the tests via copy-paste (this is what I do to debug when tests fail), then I  get the issue. But I guess that makes sense if you are saying that this is handled in the `tests/__init__.py` file, which I don't call in an interactive session..

Reply:
Ok. I'd prefer to remove this from the code, I think that copy and pasting tests into an interactive session is a useful debugging approach but we don't generally keep debugging code (like print statements) in the master branch of the repo.